src/library/VUMeter.js

```diff
@@ -0,0 +1,112 @@
+import FIFO from './FIFO.js';
+
+/**
+ * This is a VUMeter Library for creating VUMeter. It requires the 
+ * ID of canvas element where the waveform will be rendered and min
+ * decibel of for VU-meter as initialization input. minDecibel define
+ * the range of vu-meter to consider silent. When the waveform is created,
+ * it requires the analyserNode as real-time audio input data for 
+ * calculating the correct vm-meter library.
+ * 
+ * There is no library dependence, this entire VMMeter is
+ * created based on the canvas element.
+ */
+class VUMeter {
+
+  /**
+   * @constructor This is the constructor of VUMeter. Since width and height
```

Comment:
Please check the spelling. For example, "width and height *are* initialized in the constructor".